packages/vbet/vbet/vbet.py

```diff
@@ -84,11 +85,20 @@ def vbet(huc, flowlines, flowareas, orig_slope, max_slope, orig_hand, hillshade,
     vbet_network(flowlines, flowareas, cfg.OUTPUT_EPSG, vbet_network_path)
     project.add_project_vector(proj_nodes['Intermediates'], LayerTypes['VBET_NETWORK'])
 
+    # Get raster resolution as min buffer and apply bankfull width buffer to reaches
+    with rasterio.open(proj_slope) as raster:
+        t = raster.transform
+        min_buffer = (t[0] + (-t[4])) / 2
+    conversion = _rough_convert_metres_to_raster_units(proj_slope, 1)
+    reach_polygon = buffer_by_field(vbet_network_path, "BFwidth", cfg.OUTPUT_EPSG, conversion, min_buffer)
```

Comment:
The slope raster should always be in the same spatial reference as the polygon feature class, but if its not this will cause errors in the buffer. Safer to create a transform between the raster SRS and the polygon feature class SRS and reproject the min buffer distance. This will guarantee that the `min_buffer` is always in the same units as the polygon features.

Reply:
@KellyMWhitehead just a little note about GDAL transforms. They changed between GDAL 2 and GDAL 3

[I wrote a method in `shapefile.py` to help generate transforms the same way every time](https://github.com/Riverscapes/riverscapes-tools/blob/master/lib/commons/rscommons/shapefile.py#L36-L59)

You can see it in use [here: /lib/commons/rscommons/shapefile.py#L161](https://github.com/Riverscapes/riverscapes-tools/blob/master/lib/commons/rscommons/shapefile.py#L161)